### PR TITLE
Adding base for new UserManagementAPI module 

### PR DESCRIPTION
**Pull request by danielzumb**
(Also small update to README.md)

### CODE REVIEW

File: lib/restAPIs/userManagement.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const assert = require('assert');
+
+const assertParam = require('../util/assertParam');
+const RestUtil = require('../util/RestUtil');
+
+module.exports = class RestTopic extends RestUtil {
+    _getBase(){
+        return 'users';
+    }
+    getUsers(nameBegins = '', abbreviationBegins = '', offset = 0, limit = -1){
```
Comment (tiagosiebler) [inline lib/restAPIs/userManagement.js]: As discussed - optional parameters should be left blank and omitted from the request if empty:
```
getUsers(nameBegins, abbreviationBegins, offset = 0, limit = -1){
  const queryParameter = {
        offset: offset,
        limit: limit
  }
  
  if (nameBegins) {
    queryParameter.nameBegins = nameBegins;
  }
  
  if (abbreviationBegins) {
    queryParameter.abbreviationBegins = abbreviationBegins;
  }

  return this._makeRequest(this._getBase(), queryParameter, 'GET')
  .then(result => this.throwIfFailed(result, 200));
}
```

Comment (tiagosiebler) [reply lib/restAPIs/userManagement.js]: Missing the "fields" param - I would add it as last in the list

Comment (tiagosiebler) [reply lib/restAPIs/userManagement.js]: Lastly, please use indentation of 2 spaces

**Review by tiagosiebler — CHANGES_REQUESTED** [review]
Looking great!

**Review by danielzumb — COMMENTED** [review]
Self reviewed

**Review by tiagosiebler — APPROVED** [review]
Looks great!

**Pull request merged**